server/src/index.js

```diff
@@ -25,20 +25,18 @@ const {
 
 const app = express();
 
-if (NODE_ENV !== 'production') {
-    app.use(
-        cors({
-            origin(origin, callback) {
-                if (origin === APP_FRONTEND_URL) {
-                    callback(null, true);
-                } else {
-                    callback(new Error('Not allowed by CORS'));
-                }
-            },
-            credentials: true
-        })
-    );
-}
+app.use(
```

Comment:
I don't understand why only in development?

Reply:
I thought before that it will work on production without `cors`